config/config.go

```diff
@@ -1435,3 +1443,26 @@ func (c *RemoteReadConfig) UnmarshalYAML(unmarshal func(interface{}) error) erro
 	}
 	return nil
 }
+
+// AzureServiceFabricSDConfig is the configuration for Azure Service Fabric service discovery.
+type AzureServiceFabricSDConfig struct {
```

Comment:
It is doing discovery of Service Fabric applications via the Service Fabric REST API, as opposed to discovery of Azure VMs as the existing Azure discovery mechanism does.